lte/gateway/c/core/oai/tasks/mme_app/mme_app_state_manager.cpp

```diff
@@ -62,7 +62,6 @@ MmeNasStateManager::~MmeNasStateManager() {
 int MmeNasStateManager::initialize_state(const mme_config_t* mme_config_p) {
   persist_state_enabled = mme_config_p->use_stateless;
   max_ue_htbl_lists_    = mme_config_p->max_ues;
-  mme_statistic_timer_  = mme_config_p->mme_statistic_timer;
```

Comment:
There is a `MME_STATISTIC_TIMER` config value on mme.conf.template that this value reads from, maybe we should delete that as well? Are we defaulting the display interval to 10 secs now?

Reply:
Yes, we are defaulting to 10 secs. I didn't see value in changing this timer really, unless you feel strongly. I will remove this.